scripts/pylib/pytest-twister-harness/tests/device/data/mock_script.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
who runs this test and when?

Reply:
added to the "Twister TestSuite / Twister Unit Tests" check (workflow twister_tests.yaml) and is already passing on this PR.